src/rest/SearchEndpointWithDefaultCallOptions.ts

```diff
@@ -23,11 +23,12 @@ import * as _ from 'underscore';
 import { IFacetSearchRequest } from './Facet/FacetSearchRequest';
 import { IFacetSearchResponse } from './Facet/FacetSearchResponse';
 import { ExecutionPlan } from './Plan';
+import { SearchEndpoint } from '../Core';
 
 export class SearchEndpointWithDefaultCallOptions implements ISearchEndpoint {
   options: ISearchEndpointOptions;
 
-  constructor(private endpoint: ISearchEndpoint, private callOptions?: IEndpointCallOptions) {
+  constructor(private endpoint: SearchEndpoint, private callOptions?: IEndpointCallOptions) {
```

Comment:
Was the interface imported instead for circular dependency purposes?

Reply:
I made this adjustment because removing `getExportToExcelLink` from the `ISearchEndpoint` interface caused an error on line 64 in this file. Since the method still exists on `SearchEndpoint`, and the only place using `SearchEndpointWithDefaultCallOptions` passes a `SearchEndpoint`, I was able to just adjust the type here.

However, on further thought, I think it is safer to keep the `ISearchEndpoint` interface as-is and just mark the `getExportToExcelLink` as deprecated. I will revert that change and this one too,